src/s_net.c

```diff
@@ -194,7 +194,7 @@ int sockaddr_is_multicast(const struct sockaddr *sa)
     else if (sa->sa_family == AF_INET)
     {
         struct sockaddr_in *sa4 = (struct sockaddr_in *)sa;
-        return ((ntohl(sa4->sin_addr.s_addr) & 0xF0000000) == 0xE0000000);
```

Comment:
Isn't ntohl() required here due to big/little endian order?

Reply:
my line is completely wrong, I got confused between multicast and broadcast :-/ I've just reverted this.